models/Notification.php

```diff
@@ -32,12 +32,12 @@ class Notification extends AbstractModel
     protected $id;
 
     /**
-     * @var int
+     * @var string
```

Comment:
then we would have to change getter/setter too, and that would be a BC

Reply:
Getter and Setter have type hints with return string. That's why my phpstorm marks me that something is wrong here

Reply:
ahh sorry ... thought change was to int . all good.